scripts/create_gke_cluster.sh

```diff
@@ -42,23 +42,6 @@ set_pixies_cluster_config() {
   SUBNETWORK=projects/pl-pixies/regions/us-west1/subnetworks/us-west1-0
 }
 
-set_stirling_dynamic_perf_eval_cluster_config() {
-  PROJECT=pl-pixies
-  ZONE=us-west1-a
-  NETWORK=projects/pl-pixies/global/networks/dev
-  SUBNETWORK=STIRLING_PERF_DYNAMIC
-  LABELS="k8s-dev-cluster=,stirling-perf-eval-dynamic="
-}
```

Comment:
Could we consider making the dynamic subnet the default setting? Or keep this (I still use it)?

Reply:
Done!